### DISCUSSION TITLE
Trouble hosting Express app

### DISCUSSION THREAD

**Discussion by elvniv** [vercel/community#4908] [Help] [answered]
hey, i am having a problem hosting my node.js server file. when i host it locally to test, everything is fine but when i run vercel --prod then test on there. nothing is working. even when i use postman. the url is klorah.app i created a pro case and am yet to hear back :(

**Comment by amyegan** [comment]
Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that I can use to help with debugging?

**Comment by elvniv** [comment]
Unfortunately, no.

**Reply by amyegan** [reply]
I don't have enough information to determine what is going wrong for your app. If you create and share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), then I can use it to investigate the problem.

**Comment by elvniv** [comment]
could you please just point me toward like how to properly host a node
server using express thats in the same repo as a react app. i really don't
have a whole lot of time to create that

**Reply by amyegan** [reply, marked as answer]
I recommend this guide about using Express.js with Vercel: https://vercel.com/guides/using-express-with-vercel

That covers the general setup and some more advanced scenarios to consider.

**Comment by elvniv** [comment]
my guess is it has nothing to do with my code vut my file structuring

**Comment by elvniv** [comment]
Thanks, I'll check it out!
